src/plugin/notification/discord/client.ts

```diff
@@ -39,19 +44,22 @@ export class DiscordClient implements NotificationPlugin {
 
   public async notify(
     message: string,
-    data: { [key: string]: string } = {},
+    data: NotificationInputData,
     level: ENotificationLevel = ENotificationLevel.Info
   ) {
     const color = this.getMessageColor(level);
-    const fields = Object.keys(data).map((k) => ({
-      name: k,
+    const fields = Object.keys(omit(data, ["cycle"])).map((k) => ({
+      name: capitalCase(k),
       value: data[k].toString(),
     }));
     const embed = new MessageEmbed()
       .setColor(color)
       .setFooter({ text: this.hostInfo })
       .addFields(fields)
       .setTimestamp();
-    await this.client.send({ content: message, embeds: [embed] });
+    await this.client.send({
+      content: constructMessage(message, data),
```

Comment:
Yea this will break discord notifications because all relevant data were submitted through embed and the message was just `Payout report #503`.